test/testdata/deployednettemplates/recipes/devnet/gen_topology.py

```diff
@@ -0,0 +1,31 @@
+node_types = {"RN":5 }
+node_size = {"RN":"-m5d.4xl"}
+regions = [
+    "AWS-US-EAST-1",
+    "AWS-US-WEST-2",
+    "AWS-SA-EAST-1",
+    "AWS-EU-NORTH-1",
+    "AWS_AP_SOUTHEAST-2"
+]
+network="DevNet"
+f = open("topology.json", "w")
+f.write("{ \"Hosts\":\n  [")
+
+region_count = len(regions)
+first = True
+for  x in node_types:
```

Comment:
Spacing

Reply:
And should this be `node_types.items()`?

Reply:
Fixed spacing, added .keys() for clarity.